app/gui/view/component-browser/searcher-list-panel/src/lib.rs

```diff
@@ -0,0 +1,509 @@
+//! This module defines the Component Browser Panel, as sub-content of the Searcher 2.0, that shows
+//! the available components grouped by categories. It also defines the shape that the Component
+//! Browser Menu will be placed on, as this will appear as a single continuous shape.
+//!
+//! The widget is defined by the [`SearcherListPanel`].
+//!
+//! To learn more about component groups, see the [Component Browser Design
+//! Document](https://github.com/enso-org/design/blob/e6cffec2dd6d16688164f04a4ef0d9dff998c3e7/epics/component-browser/design.md).
+
+#![recursion_limit = "512"]
+// === Features ===
+#![allow(incomplete_features)]
+#![feature(negative_impls)]
+#![feature(associated_type_defaults)]
+#![feature(bool_to_option)]
+#![feature(cell_update)]
+#![feature(const_type_id)]
+#![feature(drain_filter)]
+#![feature(entry_insert)]
+#![feature(fn_traits)]
+#![feature(marker_trait_attr)]
+#![feature(specialization)]
+#![feature(trait_alias)]
+#![feature(type_alias_impl_trait)]
+#![feature(unboxed_closures)]
+#![feature(trace_macros)]
+#![feature(const_trait_impl)]
+#![feature(slice_as_chunks)]
+// === Standard Linter Configuration ===
+#![deny(non_ascii_idents)]
+#![warn(unsafe_code)]
+// === Non-Standard Linter Configuration ===
+#![allow(clippy::option_map_unit_fn)]
+#![allow(clippy::precedence)]
+#![allow(dead_code)]
+#![deny(unconditional_recursion)]
+#![warn(missing_copy_implementations)]
+#![warn(missing_debug_implementations)]
+#![warn(missing_docs)]
+#![warn(trivial_casts)]
+#![warn(trivial_numeric_casts)]
+#![warn(unused_import_braces)]
+#![warn(unused_qualifications)]
+
+
+pub mod column_grid;
+
+use ensogl_core::display::shape::*;
+use ensogl_core::prelude::*;
+
+pub use column_grid::LabeledAnyModelProvider;
+use enso_frp as frp;
+use ensogl_core::application::frp::API;
+use ensogl_core::application::Application;
+use ensogl_core::data::color;
+use ensogl_core::define_endpoints_2;
+use ensogl_core::display;
+use ensogl_core::display::object::ObjectOps;
+use ensogl_core::display::scene::Layer;
+use ensogl_core::display::shape::StyleWatchFrp;
+use ensogl_gui_component::component;
+use ensogl_list_view as list_view;
+use ensogl_scroll_area::ScrollArea;
+use ensogl_shadow as shadow;
+use ensogl_text as text;
+use ide_view_component_group as component_group;
+use ide_view_component_group::Layers;
+
+
+
+// ==============
+// === Shapes ===
+// ==============
+
+
+// === Layout Constants ===
+
+/// Scale required to covert from figma pixels measures to Enso pixel numbers.
+const DPI_SCALE: f32 = 0.5;
```

Comment:
Don't use it, please. Without it, it is easier to catch when we create something bad. For examepl, the `CONTENT_WIDTH` is set to `799.0 * DPI_SCALE`, which is bad, because on non-rtina displays it will look bad. It is hard to catch it not seeing that the thing is really a float. There are some very special legitimate cases for it, like `DIVIDER_HEIGHT`. Again, seeing numbers will be way better for debugging.